tests/test_workers/test_tasks/test_build.py

```diff
@@ -815,6 +607,9 @@ def test_skopeo_copy_fail_max_retries(mock_run_cmd):
     'force_backport, binary_image', ((True, 'binary-image:latest'), (False, None))
 )
 @pytest.mark.parametrize('distribution_scope', ('dev', 'stage', 'prod'))
+@pytest.mark.parametrize('deprecate_bundles', (True, False))
```

Comment:
Isn't the order of `imports` wrong? lines 657 and 658

Reply:
why so? Could you please elaborate on this one? Am I missing something?

Reply:
sorry, my bad. Everything is correct